src/main/java/com/aws/greengrass/clientdevices/auth/certificate/usecases/ConfigureCustomCertificateAuthority.java

```diff
@@ -25,19 +36,45 @@
  */
 public class ConfigureCustomCertificateAuthority implements UseCases.UseCase<Void, CDAConfiguration> {
     private static final Logger logger = LogManager.getLogger(ConfigureCustomCertificateAuthority.class);
-    private final CertificateManager certificateManager;
+    private final SecurityService securityService;
+    private final CertificateStore certificateStore;
 
 
     /**
      * Configure core certificate authority.
      *
-     * @param certificateManager Instance of CertificateManager
+     * @param securityService      Security Service
+     * @param certificateStore      Helper class for managing certificate
      */
     @Inject
-    public ConfigureCustomCertificateAuthority(CertificateManager certificateManager) {
-        this.certificateManager = certificateManager;
+    public ConfigureCustomCertificateAuthority(SecurityService securityService, CertificateStore certificateStore) {
+        this.securityService =  securityService;
+        this.certificateStore = certificateStore;
     }
 
+    @SuppressWarnings("PMD.AvoidCatchingGenericException")
+    private Pair<PrivateKey, X509Certificate[]> getCertificateChain(URI privateKeyUri, URI certChainUri) {
```

Comment:
This is a temporary spot for this we ideally don't have this logic inside use cases